nautobot/extras/webhooks.py

```diff
@@ -1,56 +1,52 @@
-from django.contrib.contenttypes.models import ContentType
 from django.utils import timezone
 
-
-from nautobot.core.api.utils import get_serializer_for_model
-from nautobot.core.utils.lookup import get_changes_for_model
-from nautobot.extras.models import Webhook
+from nautobot.extras.choices import ObjectChangeActionChoices
+from nautobot.extras.models import ObjectChange, Webhook
 from nautobot.extras.registry import registry
 from nautobot.extras.tasks import process_webhook
-from .choices import ObjectChangeActionChoices
 
 
-def enqueue_webhooks(instance, user, request_id, action):
+def enqueue_webhooks(object_change):
     """
     Find Webhook(s) assigned to this instance + action and enqueue them
     to be processed
     """
     # Determine whether this type of object supports webhooks
-    app_label = instance._meta.app_label
-    model_name = instance._meta.model_name
+    app_label = object_change.changed_object_type.app_label
+    model_name = object_change.changed_object_type.model
     if model_name not in registry["model_features"]["webhooks"].get(app_label, []):
         return
 
     # Retrieve any applicable Webhooks
-    content_type = ContentType.objects.get_for_model(instance)
+    content_type = object_change.changed_object_type
```

Comment:
Could probably move this up to line 14 and use it for `app_label` and `model_name`